pages/apply/[pid].js

```diff
@@ -46,6 +63,16 @@ export async function getServerSideProps({ params }) {
   };
 }
 
+export const getAdvertSchemeVersion = async (contentfulSlug) => {
+  return await axios
+    .get(
+      `${process.env.APPLICANT_BACKEND_HOST}/grant-adverts/${contentfulSlug}/scheme-version`,
```

Comment:
Can find frontend communicate to applicant backend? They run in different clusters so im not sure this will work in an env

Reply:
The find fe currently hits the admin backend to logout so should be fine :)